Monofoxe/Monofoxe.Pipeline/Monofoxe.Pipeline.csproj

```diff
@@ -1,21 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<TargetFramework>net6.0</TargetFramework>
 		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
 		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
 		<OutputPath>..\bin\$(Configuration)\</OutputPath>
 		<NoWarn>NU5100</NoWarn>
+		<Version>2.2.0</Version>
```

Comment:
Remove that, versions are set elsewhere.

Reply:
Everywhere else too.

Reply:
Oh shit forgot you do that with `$(MonofoxeVersion)`. Sorry.